opal/mca/btl/tcp/btl_tcp_endpoint.c

```diff
@@ -464,6 +464,10 @@ static void *mca_btl_tcp_endpoint_complete_accept(int fd, int flags, void *conte
         mca_btl_tcp_endpoint_event_init(btl_endpoint);
         MCA_BTL_TCP_ENDPOINT_DUMP(10, btl_endpoint, true, "event_add(recv) [endpoint_accept]");
         opal_event_add(&btl_endpoint->endpoint_recv_event, 0);
+        if( mca_btl_tcp_event_base == opal_sync_event_base ) {
```

Comment:
This code is common to the libevent and asynchronous (aka. with our own thread) progress. If we have our own thread (in which case we have our own eventset) we don't want to mess with the rest of the OMPI infrastructure and should not alter the frequency of the base eventset.